commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/CsScenarioListBuilder.java

```diff
@@ -208,10 +209,44 @@ private static CsScenarioListBuilder getPtpRoutings() {
                 subscriberPartyName,
                 publisherPartyName,
                 (CsAction) previousAction,
+                getActionTitlePtp(hasNextAction, (CsAction) previousAction),
                 componentFactory.getMessageSchemaValidator("api", "pointToPointRoutings")));
   }
 
-  private static CsScenarioListBuilder getPortSchedules() {
+  private static String getActionTitlePs(boolean hasNextAction, CsAction previousAction) {
+    if (hasNextAction) {
+      if (previousAction instanceof CsGetPortSchedulesAction) {
+        return "GetPortSchedulesSecondPage";
+      } else {
```

Comment:
```suggestion
      }
```
Code behaves the same without the `else {` part, but looks a bit cleaner. Same for line 232 and 243.